cylc/uiserver/resolvers.py

```diff
@@ -100,11 +220,42 @@ def _return(message):
             message
         ]
 
+    @classmethod
+    async def clean(cls, workflows, args, workflows_mgr, executor, log):
+        """Calls `init_clean`"""
+        response = []
+        # Convert Schema options → cylc.flow.workflow_files.init_clean opts:
+        try:
+            opts = _schema_opts_to_api_opts(args, schema=CLEAN)
+        except Exception as exc:
+            return cls._error(exc)
+        # Hard set remote timeout.
+        opts.remote_timeout = "600"
+
+        # clean each requested flow
+        for tokens in workflows:
+            clean_func = partial(_clean, tokens, opts)
+            try:
+                log.info(f'Cleaning {tokens}')
+                future = await asyncio.wrap_future(executor.submit(clean_func))
+            except Exception as exc:
+                log.exception(exc)
+                return cls._error(exc)
+            else:
+                if future == WORKFLOW_RUNNING_MSG:
+                    log.error(ServiceFileError(WORKFLOW_RUNNING_MSG))
+                    return cls._error(WORKFLOW_RUNNING_MSG)
+                else:
+                    cls._return(future)
+
+        # trigger a re-scan
+        await workflows_mgr.update()
+        return response
```

Comment:
`cls._return(future)` isn't doing anything here. It's meant to be used as the return for the method, so that the response is in the expected format. At the moment the response is empty.

Reply:
Something like
```suggestion

        # trigger a re-scan
        await workflows_mgr.update()
        return cls._return("Workflow(s) cleaned")
```